aces/Loris
# [create_timepoint] Display only visit labels related to candidate project

When creating a timepoint, the only visit labels displayed in the dropdown menu will be ones related to the candidate's project and the selected subproject using the `visit_project_subproject_rel` table. This feature is helpful for projects or instances of Loris with a long list of visit labels, so that only relevant visits are displayed, making the workflow easier for users. 

#### Testing instructions:
1. Add data to `project_subproject_rel` and `visit_project_subproject_rel` so that some project only has some of the available visit labels related to it.
2. Create a new timepoint for a candidate with the given project
3. See that it only displays relevant visit labels
